include/teb_local_planner/timed_elastic_band.h

```diff
@@ -503,6 +503,7 @@ class TimedElasticBand
    * @param max_samples maximum number of samples that should not be exceeded during resizing
    * @param fast_mode if true, the trajectory is iterated once to insert or erase points; if false the trajectory
    *                  is repeatedly iterated until no poses are added or removed anymore
+   * @param force_equal if true, the trajectory is forced to use the correct time diffs
```

Comment:
Could you please remove this parameter description? Unfortunately, I do not have write permissions in your branch, so I was not able to take over.

Reply:
Hallo @amakarow ,

Thanks for your response. The parameter description is now removed.